src/promptflow/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
   - CLI: Support `pfazure run create --resume-from <original-run-name>` to create a run resume from another run.
   - SDK: Support `pf.run(resume_from=<original-run-name>)` to create a run resume from another run.
 
+- [Executor] Added per-line logging for batch runs, stored under the `flow_logs` folder.
```

Comment:
Maybe `[Batch]` is more suitable instead of [Executor]

Reply:
Fixed.